force-app/main/default/lwc/generateRecordInputForCreate/generateRecordInputForCreate.html

```diff
@@ -0,0 +1,42 @@
+<template>
+    <lightning-card
+        title="GenerateRecordInputForCreate"
+        icon-name="standard:record"
+    >
+        <div class="slds-var-m-around_medium">
+            <template if:false={error}>
```

Comment:
I wrote it this way because I don't have a `data` property, as I decorate a function that set values to individual properties as `areaNumber`. I thought that `data` an `error` are mutually exclusive - that's what we use one template or the other (the one that shows the data **or** the error panel). Are you saying they're not mutually exclusive?

Reply:
They are not. `wire` provisions a stream of values, so it can return any of the conditions at any time and/or order. The correct was is to explicitly check for the positive result. Let's say some logic silently fails in your `data` if clause - you'd still show the UI to the user.

Reply:
Fixed